src/Dialogs/Preferences.vala

```diff
@@ -108,16 +115,33 @@ public class Workspaces.Dialogs.Preferences : Gtk.Dialog {
                 }
             }
         });
+
+        var do_last_postion = settings.get_boolean ("save-last-window-position");
+
+        var save_last_position = new Gtk.CheckButton.with_label(_ ("Save last position of window"));
```

Comment:
Can you use a label and then use a `Gtk.Switch` aligned to the right?
So it follows the rest of the app.